### ISSUE TITLE
Show subfolders in browser

### ISSUE THREAD

**Issue by niek-o** [Geoxor/amethyst#10]

**Comment by henry-lang** [comment]
How would the file browser work - when you open a folder, would it make all of the files in the explorer relative to that folder? If so, when you add a standalone track, possibly from another folder, what would happen?

**Comment by niek-o** [comment]
> How would the file browser work - when you open a folder, would it make all of the files in the explorer relative to that folder? If so, when you add a standalone track, possibly from another folder, what would happen?

Instead of loading all the files from inside the subfolders directly, opening a folder with multiple folders inside it should list the actual tree instead of just the audio files.

**Comment by Geoxor** [comment]
yes like vscode's explorer but it would use the pixel art text and folder icons (unicode characters) from the Cozette Font we are using
